hnn_core/network_builder.py

```diff
@@ -630,7 +630,7 @@ def _clear_neuron_objects(self):
         _PC.gid_clear()
 
         # dereference cell and NetConn objects
-        for gid, cell in zip(self._gid_list, self._cells):
+        for gid, _cell in zip(self._gid_list, self._cells):
```

Comment:
```suggestion
        for gid, _ in zip(self._gid_list, self._cells):
```

Reply:
I guess for this case there's no need for the zip at all if the cell isn't used in the loop? Should I just remove the zip and `self._cells`?

Reply:
I suppose we could just get rid of the zip since we the cell isn't used in the loop? Let me know if there's any reason to keep the zip.

Reply:
I don't think there is!